### ISSUE TITLE
Data populator ending message fix

### ISSUE THREAD

**Issue by gorkem-cetin** [Countly/countly-server#258]
When data populator ends, then a popup appears like: 

![ekran resmi 2016-05-04 20 05 46](https://cloud.githubusercontent.com/assets/1757036/15022298/b38eec30-1233-11e6-900d-c89087d1c074.png)

When I click OK and refresh the page as suggested, populator page is refreshed, with the following popup: 

![ekran resmi 2016-05-04 20 05 57](https://cloud.githubusercontent.com/assets/1757036/15022320/cbd20764-1233-11e6-9a0c-615388c35dc0.png)

But data is already there. 

So basically when data populator is run, then the wording would better be: 

Now your dashboard includes sample data to play with. In order to see how it looks like, wait for some time for data to settle down, and then click on Dashboard on the left.

**Comment by ar2rsawseen** [comment]
In new fixed data populator, when popup appears data is already there and upon completion
![image](https://cloud.githubusercontent.com/assets/399992/15039290/82ac0324-12b4-11e6-9abd-1cf9a3705e53.png)

when you click continue it will refresh and go to dashboard automatically.

The reason you experienced double popup it probably because you had autostart tag when moving from after app creation.

It should work properly with new data populator
